Dockerfile.rhel9

```diff
@@ -0,0 +1,32 @@
+# Build the manager binary
```

Comment:
Change the name of this file to `Dockerfile.rhel`

Reply:
Will make this change.